src/python/searchBench.py

```diff
@@ -20,16 +20,12 @@
 import re
 import random
 
-from competition import JAVA_COMMAND
 import benchUtil
 import common
 import constants
 
 PYTHON_MAJOR_VER = sys.version_info.major
 
-if '-ea' in sys.argv:
```

Comment:
Why is this the right way to fix the circular dependency?

Reply:
The circular dependency comes from line

```
from competition import JAVA_COMMAND
```

and the problem is that `competition.py` itself imports this file, `searchBench.py`.

We can also fix support for `'-ea'` argument, but given that it never really worked makes me think that noone uses the argument, so it is easier to remove.

Reply:
> but given that it never really worked 

Just to expand on that, this file uses `constants.JAVA_COMMAND` in all other places.

Reply:
Got it, thank you!